test/HostActivationTests/GivenThatICareAboutStandaloneAppActivation.cs

```diff
@@ -154,20 +183,17 @@ private static void ReplaceTestProjectOutputHostInTestProjectFixture(TestProject
         {
             var dotnet = testProjectFixture.BuiltDotnet;
 
-            var testProjectHost = testProjectFixture.TestProject.AppExe;
             var testProjectHostPolicy = testProjectFixture.TestProject.HostPolicyDll;
             var testProjectHostFxr = testProjectFixture.TestProject.HostFxrDll;
 
-            if (!File.Exists(testProjectHost) || !File.Exists(testProjectHostPolicy))
+            if (!File.Exists(testProjectHostPolicy))
             {
                 throw new Exception("host or hostpolicy does not exist in test project output. Is this a standalone app?");
             }
 
-            var dotnetHost = Path.Combine(dotnet.GreatestVersionSharedFxPath, $"dotnet{testProjectFixture.ExeExtension}");
             var dotnetHostPolicy = Path.Combine(dotnet.GreatestVersionSharedFxPath, $"{testProjectFixture.SharedLibraryPrefix}hostpolicy{testProjectFixture.SharedLibraryExtension}");
-            var dotnetHostFxr = Path.Combine(dotnet.GreatestVersionSharedFxPath, $"{testProjectFixture.SharedLibraryPrefix}hostfxr{testProjectFixture.SharedLibraryExtension}");
+            var dotnetHostFxr = Path.Combine(dotnet.GreatestVersionHostFxrPath, $"{testProjectFixture.SharedLibraryPrefix}hostfxr{testProjectFixture.SharedLibraryExtension}");
```

Comment:
actually this bug is still functional in core-setup repo, i could not find a way to get the project.json tooling with the fix, We should remove it when we upgrade our project system completely